gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicationMetaData.java

```diff
@@ -0,0 +1,41 @@
+package gobblin.data.management.copy.replication;
+
+import com.google.common.base.Optional;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * Class used to represent the meta data of the replication
+ * @author mitu
+ *
+ */
+
+@AllArgsConstructor
```

Comment:
You can use `@Data` instead, it combines `@AllArgsConstructor`, `@Getter`, and generates a `toString()` method very similar to the one you implemented.